openshift/cm.yml

```diff
@@ -77,219 +83,62 @@ objects:
   data:
     properties: |
       # log level
-      logging.level.root=WARN
-      logging.level.org.springframework=INFO
-      logging.level.org.springframework.security=DEBUG
-      logging.level.com.atlassian=INFO
       logging.level.com.atlassian.crowd=${PROV_APP_LOG_LEVEL_ATLASSIAN_CROWD}
       logging.level.org.opendevstack=${PROV_APP_LOG_LEVEL_OPENDEVSTACK}
 
       # log file
       logging.file.path=${LOGGING_FILE_PATH}
 
       # atlassian API calls take sooo long
-      server.servlet.session.timeout=240
-      server.connection-timeout=90000
-      spring.session.timeout=240
-      spring.mvc.async.request-timeout=90000
-      
-      # server port
-      server.port=8080
-      server.use-forward-headers=true
-
       global.keyuser.role.name=${idmanager.group.opendevstack-administrators}
 
-      project.template.default.key=default
-
-      # project types, these two are configured in the src/main/resources/application.properties
-      project.template.key.names=${project.template.default.key},kanban
-
       # JIRA properties
       jira.uri=${JIRA_URL}
-      jira.api.path=/rest/api/latest
-      jira.permission.filepattern=classpath*:permission-templates/jira.permission.*
-
-      # JIRA default project templates
-      jira.project.template.key=com.pyxis.greenhopper.jira:gh-scrum-template
-      jira.project.template.type=software
-
-      # kanban project type
-      jira.project.template.key.kanban=com.pyxis.greenhopper.jira:gh-kanban-template
-      jira.project.template.type.kanban=software
 
       # Confluence properties
       confluence.uri=${CONFLUENCE_URL}
-      confluence.api.path=/rest
-      confluence.json.rpc.api.path=/rpc/json-rpc/confluenceservice-v2
-      confluence.permission.filepattern=classpath*:permission-templates/confluence.permission.*
 
       # Bitbucket properties
       bitbucket.uri=${BITBUCKET_URL}
-      bitbucket.api.path=/rest/api/1.0
-      bitbucket.repository.pattern=%s-%s
       bitbucket.default.user.group=${PROV_APP_CROWD_USER_GROUP}
       bitbucket.technical.user=${CD_USER_ID}
-      bitbucket.opendevstack.project=opendevstack
```

Comment:
Could you not remove this, but instead parameterise it by setting it to https://github.com/opendevstack/ods-core/blob/master/configuration-sample/ods-core.env.sample#L19? Thanks!